src/ui/public/field_editor/field_editor.js

```diff
@@ -37,7 +39,7 @@ uiModules
 
       self.docLinks = docLinks;
       getScriptingLangs().then((langs) => {
-        self.scriptingLangs = langs;
+        self.scriptingLangs = _.intersection(langs, ['expression', 'painless']);
```

Comment:
I kind of like the flexibility of the existing functions. If you're ok with it, let's wait and see if we need a `filterSupportedScriptingLangs` elsewhere before creating it.